lego-mindstorms-pieces.py

```diff
@@ -174,11 +178,38 @@ def order(shop=None, browser=None, lego_set=None, order_list=None, username=None
 
     order_list = order_list.split(',')
 
-    shop_url = 'https://wwwsecure.us.lego.com/{shop}/service/replacementparts/order'.format(shop=shop)
-    browser = Chrome() if browser == 'chrome' else Firefox()
+    print("Using Selenium version : ",webdriver.__version__)
+
+    shop_url = 'https://wwwsecure.us.lego.com/{shop}/service/replacementparts/sale'.format(shop=shop)
+
+    print("Browser URL : ",shop_url)
```

Comment:
I'm not sure this will work across Python 2 and 3 versions. Please use string formatting instead, e.g.
``` python
print("Browser URL: {url}".format(url=shop_url))
```